.github/workflows/format.yml

```diff
@@ -33,9 +33,19 @@ jobs:
         - name: Run Black
           run: black ./ --config pyproject.toml
 
+        - name: Check Black formatting
+          uses: psf/black@stable
+          with:
+            options: "--check --verbose"
+            src: "./"
+            # use_pyproject: true, should be used but causing issues so commenting for now
```

Comment:
Can you provide more details as to what's up here? We don't have to resolve the issue in this PR, but any comments might help the next person. Perhaps we should make an issue for this?